test/Microsoft.DotNet.ApiCompatibility.Tests/Rules/AssemblyIdentityMustMatchTests.cs

```diff
@@ -196,8 +197,10 @@ public void RetargetableFlagSet(bool strictMode)
             string leftAssembly = SymbolFactory.EmitAssemblyFromSyntax(syntax, publicKey: _publicKey);
             string rightAssembly = SymbolFactory.EmitAssemblyFromSyntax(syntax);
 
-            IAssemblySymbol leftSymbol = new AssemblySymbolLoader().LoadAssembly(leftAssembly);
-            IAssemblySymbol rightSymbol = new AssemblySymbolLoader().LoadAssembly(rightAssembly);
+            ILog logger = new ConsoleLog(MessageImportance.High);
```

Comment:
We already define a log instance for the rule factory: https://github.com/dotnet/sdk/blob/fd87d94ae05fe5af58bc87de7d06cae8dbed9ebf/test/Microsoft.DotNet.ApiCompatibility.Tests/Rules/AssemblyIdentityMustMatchTests.cs#L17

Please use that instead.

Reply:
Seem you're talking about the `new SuppressibleTestLog()` being passed to `AssemblyIdentityMustMatch`.

I assume you mean I should extract that instance, store it in a field, and reuse that everywhere?

Reply:
If not, then I'm struggling to understand how to obtain the logger out of `s_ruleFactory`, if that's possible.

Reply:
> I assume you mean I should extract that instance, store it in a field, and reuse that everywhere?

Yes